src/RecoveryServices/RecoveryServices/Az.RecoveryServices.psd1

```diff
@@ -66,7 +66,8 @@ RequiredAssemblies = '.\AutoMapper.dll',
                '.\Microsoft.Azure.Management.RecoveryServices.Backup.dll', 
                '.\Microsoft.Azure.Management.RecoveryServices.dll', 
                '.\Microsoft.Azure.Management.RecoveryServices.SiteRecovery.dll', 
-               '.\System.Configuration.ConfigurationManager.dll'
+               '.\System.Configuration.ConfigurationManager.dll',
+               '.\TimeZoneConverter.dll'
```

Comment:
@siddharth7 we should be cautious about external dependencies that we reference in our code:

- It may not be signed
- It may reference additional packages that we cannot allow
- It may reference different versions of packages that we currently consume in our modules

Is there a reason we're using this package over what's built into .NET?

Reply:
@cormacpayne This reason for using this library is because the difference in timezone formats for Windows and unix machine that hasn't been solved by dotnet as stated in this blog. https://dejanstojanovic.net/aspnet/2018/july/differences-in-time-zones-in-net-core-on-windows-and-linux-host-os/